.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,18 @@
+---
+name: Bug Report 🐞
+about: Something isn't working as expected? Here is the right place to report.
+labels: "type: bug"
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue will be closed. This info allows Metasploit maintainers to diagnose (and fix!) your issue as quickly as possible.
+
+  Useful Links:
+  - Wiki: https://github.com/rapid7/metasploit-framework/wiki
+
+  Before opening a new issue, please search existing issues: https://github.com/rapid7/metasploit-framework/issues
+-->
+
 ## Steps to reproduce
```

Comment:
Would it be worthwhile adding a section for what tutorial/documentation are you following?

Reply:
Couldn't hurt!